dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/DateUtils.java

```diff
@@ -480,6 +480,21 @@ public static Date getLastDayOfMonth(Date date) {
         return cal.getTime();
     }
 
+    /**
+     * get a month ago
+     *
+     * @param date date
+     * @return get a month ago
+     */
+    public static Date getMonthAgo(Date date) {
```

Comment:
can we reuse the function name `addMonths` in this util?

Reply:
done.